tools/linux/uio/uio.mk

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright 2024, UNSW
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# This Makefile snippet builds block UIO driver
+#
+
+ifeq ($(strip $(VMM_TOOLS)),)
```

Comment:
```suggestion
ifeq ($(strip $(LIBVMM_TOOLS)),)
```

Reply:
Also why can't it be `LIBVMM_TOOLS ?= ../../`

Reply:
You can't use relative paths in snippets as the makefiles get copied and executed from an arbitrary build directory

Reply:
Will rename variable though

Reply:
Renamed